coderd/util/ptr/ptr.go

```diff
@@ -0,0 +1,28 @@
+// Package ptr contains some utility methods related to pointers.
+package ptr
+
+import "golang.org/x/exp/constraints"
+
+type number interface {
+	constraints.Integer | constraints.Float
+}
+
+// Ref returns a reference to v.
+func Ref[T any](v T) *T {
+	return &v
+}
+
+// Deref dereferences v. Opposite of Ptr.
+func Deref[T any](v *T) T {
+	return *v
+}
```

Comment:
I'd argue against having Deref in this package as it's not checked. Consumers should check and then `*v` themselves